### ISSUE TITLE
Tunnel with default route: no access to local network.

### ISSUE THREAD

**Issue by Chris56** [zaneschepke/wgtunnel#446]
When a tunnel defines a default route, i.e. AllowedIPs = 0.0.0.0/0
The access to the local network is not possible anymore.
This is different on Linux desktop and at least the tunnel packets themselves MUST run throuch the local net.
No way to reach the local resources

**Smartphone (please complete the following information):**

- Device: any
- Android Version: serveral, e.g.13
- App Version [e.g. 3.3.3] 3.6.0
- Backend: [e.g. Kernel, Userspace]

**To Reproduce**
Just define a default route in conf file.

**Expected behavior**
Apart from the default route throght the tunnel an additional route to the local network.

**Comment by zaneschepke** [comment]
Hello! Please see the [FAQ](https://zaneschepke.com/wgtunnel-docs/faq.html#how-do-i-exclude-my-local-network-from-the-tunnel-so-i-can-access-my-local-services-and-devices) for how to resolve this. 

In the near future, I'm looking to add settings to the config screen to exclude private IPs from the tunnel w/custom ranges, but (for now) the directions in the FAQ are the best way to resolve this.

**Comment by Chris56** [comment]
Thanks for your reply.
I used this as an awkward workaround before.
But apart from splitting 0.0.0.0/0 into a lengthy list: every time the android device moves to another local network, you will have to change the config. However, the config file should be agnostic of the network the device happens to be in.
The (quite simple) Linux solution is creating a different routing table for the new default route, which is only being used, in case there is no local route defined for the particular destination ip: "ip -4 rule add table main suppress_prefixlength 0"

**Comment by Chris56** [comment]
I propose to leave it open as long as the additional settings aren't yet implemented and close it only afterwards. That way people will know about the implementation having taken place.

**Comment by zaneschepke** [comment]
Sounds good. 

I've changes to name to better reflect what feature this one is requesting in order to be closed.

**Comment by dkoppenh** [comment]
Looks like my feature request #402. Simple toggle switch to allow LAN traffic.

**Comment by Chris56** [comment]
You are right. The only reason, why I opened a "BUG" was, because the configuration file should know anything about the devices paticular network.

**Comment by dkoppenh** [comment]
I like the built in allowed IP calculator, thanks.

Could an option be added to a tunnel that allows LAN traffic, regardless of the subnet wgtunnel is currently on?

Thanks, 
David

**Comment by zaneschepke** [comment]
I think this one is effectively already resolved with the peer actions when you edit a tunnel. There is an three dot menu action to exclude LAN that will add the appropriate allowedIps to bypass private addresses from the tunnel. It is also private IP DNS aware so it won't accidentally exclude your tunnel's private IP DNS servers. 

Marking as completed for now.
